soh/soh/util.cpp

```diff
@@ -274,6 +274,203 @@ std::vector<std::string> itemNames = {
     "Deku Nut Upgrade (40)",
 };
 
+std::vector<std::string> randomizerItemNames = {
```

Comment:
Since this list is essentially static and wont change when running, I think it would be better to use a std::array so we can use `RG_MAX` as the array size and force a compile error when even a new `RG_` value is added but the list was not updated.

```suggestion
std::array<std::string, RG_MAX> randomizerItemNames = {
```